src/components/dataTable/DataTableRow.tsx

```diff
@@ -0,0 +1,26 @@
+export type DataTableRowProps = {
```

Comment:
it's a good practice to have all our related types in types.tsx file

Reply:
nice, will change

Reply:
actually, you will want to use a shared types file, when there are multiple consumer for the types, it aim to assist with central of the import, and to avoid cycle references.

When it is just about a props of a component, i prefer to place the props type right above it